api/dss.yaml

```diff
@@ -1492,6 +1576,9 @@ components:
           format: date-time
           description: End time of service.  RFC 3339 format, per OpenAPI specification.
           type: string
+        version:
+          description: 'Version string required for replacing existing IdentificationServiceArea'
```

Comment:
nit: let's be stylistically-consistent and put spaces in Identification Service Area

Reply:
ack, however I removed description to fix the above issue.